### ISSUE TITLE
ERROR    All families are singleton families, No Ks can be calculated - RBH

### ISSUE THREAD

**Issue by dfmoralesb** [heche-psb/wgd#29]
Hello!

I've been using wgd dmd for whole paranome and then wgd ksd get the Ks distribution of single species and everything works great. But when trying to run dmd and ksd for RBHs, I got the same error below no mater what samples I use.

`ERROR    All families are singleton families, No Ks can be calculated ` 

I used all defaults parameters as in the examples. I there any parameter that I'm missing? I'm using wgd v2.0.30 

wgd dmd sequence1 sequence2
wgd ksd families sequence1 sequence2

Thanks for you help!
Best,
Diego

**Comment by heche-psb** [comment]
Hi, it seems that the gene families provided for `wgd ksd` has only singleton families. Is it the case?

**Comment by dfmoralesb** [comment]
Yes, that's the case. So, I don't understand why `wgd dmd`  outputs only singleton families.

**Comment by heche-psb** [comment]
May you share me with your sequence file? It has something to do with the seq itself.

**Comment by dfmoralesb** [comment]
Here are the CDS files for two species. I tried with many others as well, but I got the same error. Thanks! 
[CDS.zip](https://github.com/heche-psb/wgd/files/14950527/CDS.zip)

**Comment by heche-psb** [comment]
Hi, I spotted the issue, which is due to the missing of index (like GF00000001) in the RBH gene families file. I have fixed the issue and updated it in `v2.0.31`. May you install the latest version and try with this command for the `wgd dmd` step again? (note that the only difference is the addition of the flag `-of`)

`$wgd dmd Cypripedium_macranthos_var_rebunense.cds.fa Vanilla_shenzhenica.cds.fa -of`
